common/test/acceptance/pages/lms/course_nav.py

```diff
@@ -126,6 +126,7 @@ def go_to_sequential(self, sequential_title):
             # Convert the list index (starts at 0) to a CSS index (starts at 1)
             seq_css = "ol#sequence-list>li:nth-of-type({0})>a".format(seq_index + 1)
             self.q(css=seq_css).first.click()
+            self.wait_for_ajax()
```

Comment:
Please add a comment to the effect that the click triggers an ajax event. I suspect that we have too many wait_for_ajax calls sprinkled around "just in case" and so at least for the new ones it will be more clear where they are intentional.